fuzzydir.lua

```diff
@@ -124,10 +124,17 @@ end
 function fast_readdir(path)
     if powershell_version >= 3 then
         return call_command({
-            "powershell.exe",
+            "powershell",
             "-NoProfile",
             "-Command",
-            "& { Get-ChildItem -LiteralPath FileSystem::\"" .. path .. "\" -Directory | foreach { $_.Name } }",
+            [[
+            $dirs = Get-ChildItem -LiteralPath ]] .. string.format("%q", path) .. [[ -Directory

```

Comment:
Missing the `FileSystem::` part right before the path, will break with direct access to network shares (paths like `\\server\name\directory`) if I remember correctly the results from my previous tests.

The resulting string should look like `FileSystem::"path"`

Reply:
Or not. Retested this, seems to be working fine for some reason.